tests/test_specsynthase.py

```diff
@@ -1,17 +1,43 @@
-import os
-from os import path
+import pytest
 import yaml
+from os import path
 
 from specsynthase.specbuilder import SpecBuilder
 
 
-def test_main():
-    spec = SpecBuilder()
+def _get_base_dir():
+    return path.join(path.abspath(path.dirname(__file__)), "fixtures", "petstore")
+
+
+def _check_dicts(l, r):
+    for k, v in l.items():
+        assert (k in r)
+        if isinstance(v, dict):
+            _check_dicts(v, r[k])
+
+
+def test_build_fluent():
```

Comment:
This is fluent due to way that is possible to call add_spec and others functions that returns "self":

` SpecBuilder().add_spec(path.join(base_dir, "security.yml")).add_spec(path.join(base_dir, "base.yml"))`